server/api/store.go

```diff
@@ -222,10 +222,18 @@ func (h *storeHandler) SetState(w http.ResponseWriter, r *http.Request) {
 		h.rd.JSON(w, http.StatusBadRequest, "invalid state")
 		return
 	}
+	var err error
+	switch metapb.StoreState(state) {
```

Comment:
If the `state` is `tombstone`, I think we can return the response that `tombstone` is not supported to be set anymore.